docs/quick_start/index.md

```diff
@@ -8,7 +8,7 @@ LeanCode CoreLibrary offers flexibility to suit different needs. Whether you wan
 
 ## Setting up the template
 
-The `lncdproj` template, provided by LeanCode CoreLibrary, is a comprehensive template designed for modern .NET application development, featuring CQRS as client-facing APIs, PostgreSQL for database management, and Kratos for identity management. It integrates MassTransit with RabbitMQ for efficient communication and includes tools to aid testing. Additionally, the template incorporates LeanPipe for real-time SignalR notifications, Blob Storage for managing large data, and resources like Seq for logging, Jaeger for tracing, and Traefik as an application proxy, collectively laying a solid foundation for building efficient .NET applications.
+The `lncdproj` template, provided by LeanCode CoreLibrary, is a comprehensive template designed for modern .NET application development, featuring CQRS as client-facing APIs, PostgreSQL for database management, and Kratos for identity management. It integrates MassTransit with RabbitMQ for efficient communication and includes tools to aid testing. Additionally, the template incorporates LeanPipe for real-time SignalR notifications, Blob Storage for managing large data, and resources like Seq for logging, Jaeger for tracing, and Traefik as an application proxy, collectively laying a solid foundation for building efficient .NET applications in a development setting.
```

Comment:
This does not really solve the problem. What is "in a development setting"? LeanPipe? Blob Storage? We should focus here on the template from the perspective of deployed application, leaving dev setting to the section below that describes it well already.

Reply:
Okay, included here only things that are common for both dev and production setting.